pages/docs/platform/monitor/insights.mdx

```diff
@@ -0,0 +1,209 @@
+
+For more details on the event format, see the [Inngest Event Format documentation](https://www.inngest.com/docs/features/events-triggers/event-format).
+
+### Data Retention
+
+You can query events from **up to 30 days in the past**. Older events are not available through Insights.
+
+### Result Limits
+
+- Current page limit: **1000 rows**
+- Future updates will support larger result sets through async data exports
+- Pagination support is planned for future releases
+
+## SQL Support
+
+Insights is built on ClickHouse, which provides powerful SQL capabilities with some differences from traditional SQL databases.
+
+![Sql Editor View](/assets/docs/platform/monitor/insights/insights_query_results.png)
+
+### Supported Functions
+
+#### Arithmetic Functions
+Basic mathematical operations and calculations.
+[View ClickHouse arithmetic functions documentation](https://clickhouse.com/docs/sql-reference/functions/arithmetic-functions)
+
+#### String Functions
+String manipulation and search capabilities.
+- [String search functions](https://clickhouse.com/docs/sql-reference/functions/string-search-functions)
+- [String manipulation functions](https://clickhouse.com/docs/sql-reference/functions/string-functions)
+
+#### JSON Functions
+Essential for working with `event_data` payloads.
+[View ClickHouse JSON functions documentation](https://clickhouse.com/docs/sql-reference/functions/json-functions)
+
+#### Date/Time Functions
+For analyzing event timing and patterns.
+[View ClickHouse date/time functions documentation](https://clickhouse.com/docs/sql-reference/functions/date-time-functions)
+
+#### Other Supported Function Categories
+- [Logical functions](https://clickhouse.com/docs/sql-reference/functions/logical-functions)
+- [Rounding functions](https://clickhouse.com/docs/sql-reference/functions/rounding-functions)
+- [Type conversion functions](https://clickhouse.com/docs/sql-reference/functions/type-conversion-functions)
+- [Functions for nulls](https://clickhouse.com/docs/sql-reference/functions/functions-for-nulls)
+- [ULID functions](https://clickhouse.com/docs/sql-reference/functions/ulid-functions)
+
+### Aggregate Functions
+
+The following aggregate functions are supported:
+
+| Function | Description |
+|----------|-------------|
+| `ARRAY_AGG()` | Aggregates values into an array* |
+| `AVG()` | Calculates average |
+| `COUNT()` | Counts rows |
+| `MAX()` | Finds maximum value |
+| `MIN()` | Finds minimum value |
+| `STDDEV_POP()` | Population standard deviation |
+| `STDDEV_SAMP()` | Sample standard deviation |
+| `SUM()` | Calculates sum |
+| `VAR_POP()` | Population variance |
+| `VAR_SAMP()` | Sample variance |
+| `median()` | Finds median value |
+
+> **\*Note on ARRAY_AGG**: Due to a current serialization bug, you need to convert arrays to strings using `toString(ARRAY_AGG(column_name))`.
+
+[View complete ClickHouse aggregate functions documentation](https://clickhouse.com/docs/sql-reference/aggregate-functions/reference)
```

Comment:
We don't support any functions on this list other than the 10 or so from the table above, so I think we should either leave this link out or link each row to the specific ClickHouse docs page